app/src/main/res/values/styles.xml

```diff
@@ -7,6 +7,11 @@
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
+
+    <style name="ErrorTextAppearance">
```

Comment:
Going back to my comment about managing lint errors. It appears to me that the lint error could be resolved by simply introducing this style under the community variant resources.

Reply:
Ah yep that fixed it. Sorry, somehow I totally overlooked the ```includedVariants``` line and thought it was obviously being used in the community variant 😆. Good catch 👍